src/table.rs

```diff
@@ -208,7 +254,7 @@ where
     match fat_type {
         FatType::Fat12 => {
             fat.write_u8(media)?;
-            fat.write_u16_le(0xFFFF)?;
+            fat.write_u16_le(0xFFF)?;
```

Comment:
I think this is not correct, it makes second entry 0x0FF instead of 0xFFF (each entry is 12 bit and and here we write 4 bits of first entry and 12 of the second)

Reply:
Ah, you're right.